src/main/app/src/pages/BrowsePage.js

```diff
@@ -70,9 +106,14 @@ const BrowsePage = ({ links, activityType, updateActivityType, updateActivity, u
                 )
               })}
             </div>
+            {/** Pagination menu. */}
+            <TablePagination component="div" className="mx-auto" count={filteredLinks.length} page={pageNumber} onChangePage={(evt, page) => updatePageNumber(page)}
+              rowsPerPage={activitiesPerPage} onChangeRowsPerPage={handleActivitiesPerPageChange} />
           </div>
         </div>
       </div>
+      {/** Show loading indicator once user presses the form submit button. */}
```

Comment:
ditto.

Reply:
I think this is Steve's code so I'll let him change it on his commit! :)